optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/api/PlannerBenchmark.java

```diff
@@ -25,13 +27,17 @@
 
     /**
      * Run all the single benchmarks and create an overview report.
+     *
+     * @return never null, path to the directory in which the benchmark results are stored
      */
-    void benchmark();
+    Path benchmark();
 
     /**
      * Run all the single benchmarks, create an overview report
      * and show it in the default browser.
+     *
+     * @return never null, path to the directory in which the benchmark results are stored
      */
-    void benchmarkAndShowReportInBrowser();
+    Path benchmarkAndShowReportInBrowser();
```

Comment:
Technically, this is backwards incompatible. Practically, it will not do any harm.
It should still be noted in the upgrade recipe, which I will do after this PR is approved.